src/js/GaCesium.js

```diff
@@ -115,7 +125,61 @@ var GaCesium = function(map, gaPermalink, gaLayers, gaGlobalOptions,
     scene.screenSpaceCameraController.inertiaSpin = 0.5;
     scene.screenSpaceCameraController.inertiaTranslate = 0.8;
     scene.screenSpaceCameraController.inertiaZoom = 0.9;
+    scene.screenSpaceCameraController.minimumZoomDistance = 2;
+
     enableOl3d(cesiumViewer, enabled);
+
+    // Tileset 3D
+    var tileset3d = [
+      'ch.swisstopo.swisstlm3d.3d'
+    ];
+
+    var primitives = [];
+    var params = gaPermalink.getParams();
+    var pTileset3d = params['tileset3d'];
+
+    tileset3d = pTileset3d ? pTileset3d.split(',') : tileset3d;
+
+    tileset3d.forEach(function(tileset3dId) {
+      if (tileset3dId) {
+        var tileset = gaLayers.getCesiumTileset3DById(tileset3dId);
+        if (/names3d\.3d/.test(tileset3dId)) {
+          tileset.style = new Cesium.Cesium3DTileStyle({
+            show: true,
+            color: 'rgb(255, 255, 255)',
+            outlineColor: 'rgb(0, 0, 0)',
+            outlineWidth: 3,
+            labelStyle: 2,
+            font: "'24px arial'"
+
+          });
+          /*tileset.style = new Cesium.Cesium3DTileStyle({
+            color: {
+              conditions: [
+                ['${Height} >= 100', 'color("orange", 0.5)'],
+                ['${Height} < 100', 'color("blue")'],
+                ['true', 'color("blue")']
+              ]
+            },
+            show: '${Height} > 0',
+            meta: {
+              description:'"Building id ${id} has height ${Height}."'
+            }
+          });*/
+        }
+        if (tileset) {
+          scene.primitives.add(tileset);
+          primitives.push(tileset);
```

Comment:
that was my first idea but we don't know what could be inside `scene.primitives` in the future and how differentiate each tileset to remove to the others so, to be sure, I use another array.